test/foundry-tests/reactors/RelayOrderReactorTest.t.sol

```diff
@@ -176,6 +192,21 @@ contract RelayOrderReactorTest is GasSnapshot, Test, PermitSignature, DeployPerm
         reactor.execute(signedOrder, filler);
     }
 
+    function test_execute_noInputs_reverts_SignatureExpired() public {
```

Comment:
nit: can you name this test_execute_reverts ? its nice for regex matching and for consistency